.github/workflows/dependency-graph.yml

```diff
@@ -0,0 +1,12 @@
+name: Update Dependency Graph
+on:
+  push:
+    branches:
+      - main
+jobs:
+  dependency-graph:
+    name: Update Dependency Graph
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: scalacenter/sbt-dependency-submission@v2
```

Comment:
Okay... so what are going to be the developer-visible effects of this action? an email notification if some of the dependencies is flagged as vulnerable?

Reply:
https://docs.github.com/en/code-security/dependabot/dependabot-alerts/about-dependabot-alerts#access-to--dependabot-alerts

> By default, we notify people with admin permissions in the affected repositories about new Dependabot alerts

We can configure / disable notifications. 

Also:
> Dependabot alerts are displayed on the Security tab for the repository and in the repository's dependency graph. The alert includes a link to the affected file in the project, and information about a fixed version.

Reply:
looks like e-mail by default - to admins + we have it visible on the Security tab